src/ast/codegen_llvm.cpp

```diff
@@ -2966,8 +2954,26 @@ void CodegenLLVM::optimize()
 std::unique_ptr<BpfOrc> CodegenLLVM::emit(void)
 {
   assert(state_ == State::OPT);
-  orc_->compileModule(move(module_));
+  orc_->compile(move(module_));
   state_ = State::DONE;
+
+#ifdef LLVM_ORC_V2
```

Comment:
so because I cannot figure out how to tell llvm to materialize/codegen the whole module right away this kludge triggers the compile by calling lookup. It looks like llvm compiles the whole module when you ask for one func, so we can exit early.
For this to work the probe_ variables need to be public in bpftace. 

I have some ideas to clean this up in the future, by dumping the function list in the module and then asking it to materialize that. I also want to move the probe and attach logic out of bpftrace to a separate module, so Ill tackle both in a follow up.

Unless someone knows how to emit the code right away without creating a `MaterializationResponsibility` that list all symbol we need.